.gitignore

```diff
@@ -30,4 +30,4 @@ pom.xml.versionsBackup
 .metals
 .vim
 shell.nix
-tags
+tags
```

Comment:
This change will still cause a ~2 hour long full PR build instead of build for this new add-on which will take only a few minutes.

Reply:
Updated.  Not sure where that change came from...  Hope builds go faster now.

Reply:
The change was a missing new line at the end of the file... now you've also removed another line. 🙃

Reply:
Yes it builds a lot faster now!